scipy/integrate/_quadpack_py.py

```diff
@@ -62,28 +66,36 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
         Non-zero to return a dictionary of integration information.
         If non-zero, warning messages are also suppressed and the
         message is appended to the output tuple.
+        
+        .. deprecated:: 1.11.0
+           Parameter `full_output` is deprecated and will be removed in
+           version 1.13.0. When `full_output` is unspecified, the contents
+           of ``infodict`` are provided as attributes of the result object.
     complex_func : bool, optional
         Indicate if the function's (`func`) return type is real
         (``complex_func=False``: default) or complex (``complex_func=True``).
         In both cases, the function's argument is real.
-        If full_output is also non-zero, the `infodict`, `message`, and
-        `explain` for the real and complex components are returned in
-        a dictionary with keys "real output" and "imag output".
+        The `infodict`, `message`, and
+        `explain` for the real and complex components are
+        a dictionaries with keys "real output" and "imag output".
```

Comment:
Whatever happens, it was supposed to be "real" and "imag".
```suggestion
       dictionaries with keys "real" and "imag".
```

Reply:
Thanks, typo!